### DISCUSSION TITLE
Tabs instead of spaces

### DISCUSSION THREAD

**Discussion by screendriver** [renovatebot/renovate#12613] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

29.4.3

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

14.4.1-ee

### Describe the bug

Since five days Renovate is not respecting our Prettier config anymore and uses Tabs instead of Spaces when its updating our `package-lock.json`

![2021-11-11 16 22 04 gitlab p7s1 io 70743eb96181](https://user-images.githubusercontent.com/149248/141323646-6ae06642-9745-401e-902e-c4358d0d1822.png)

Since we are validating all files if they met our styles all of our pipelines started to fail because of that. Additionally our Renovate instance also updates itself whenever you release a new version so I cannot directly tell when this issue exactly started in which version but it started 5 days ago.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Converted this to a discussion, because it's lacking a reproduction plus I suspect it's due to npm and not Renovate. Renovate generally relies on npm itself to format the lock files.

Did you change Renovate version?
Can you find out which npm version was used previously and now?

**Reply by rarkins** [reply]
https://github.com/npm/cli/issues/3948

**Reply by screendriver** [reply]
We are using npm `7.21.1` which is shipped with Node.js `16.9.1`. So this did not change since over a month because the whole Docker image in the pipeline is pinned to exactly this versions. The only moving part here is the dependency to Renovate so I thought it must be related to Renovate 🤔

**Reply by rarkins** [reply]
You're right, I found the change which caused the bug. Fix in #12715

**Reply by screendriver** [reply]
Nice! Thank you 🎉
